src/Compilers/CSharp/Test/Emit3/RefStructInterfacesTests.cs

```diff
@@ -25316,6 +26445,147 @@ public interface I1 : System.Collections.IEnumerable
                 );
         }
 
+        [Fact]
+        public void ObjectCreation_LanguageVersion_05()
+        {
+            var text1 = @"
+public ref struct S : I1
+{
+    public int P {get;set;}
+    public void Add(int x) => P += x;
+    System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() => null;
+}
+
+public interface I1 : System.Collections.IEnumerable
+{
+    public void Add(int x);
+}
+";
+            var comp1 = CreateCompilation(text1, targetFramework: s_targetFrameworkSupportingByRefLikeGenerics);
+
+            var text2 = @"
+class Program
+{
+    static T Test<T>()
+        where T : I1, new(), allows ref struct
+    {
+        return new T() { 100, 20, 3 };
+    }
+
+    static void Main()
+    {
+        System.Console.Write((new S() { 200, 40, 6 }).P);
+    }
```

Comment:
Was the intent to run the code for the success cases?

Reply:
> Was the intent to run the code for the success cases?

No. Execution is covered by other tests. I think `ObjectCreation_05` in this particular case